src/js/plugins/plugin.search.js

```diff
@@ -412,7 +412,7 @@ BookReader.prototype.updateSearchHilites2UP = function() {
       const pageIsInView = jQuery.inArray(pageIndex, this.displayedIndices) >= 0;
 
       if (pageIsInView) {
-        if (box.div === null) {
+        if (!box.div) {
```

Comment:
Because apparently `undefined == null` is true only with double equality comparison 🤷‍♂